build/azure-pipelines/build-product.yml

```diff
@@ -7,7 +7,7 @@ pr: none
 
 parameters:
   - name: PushToGithub
-    default: true
+    default: false

```

Comment:
this has to be true for the automated build to run.

Reply:
Had changed this to false to test the changes. Have reset it to true.